acceptance/suites/tests/010-puppetserver-cli/subcommand/gem.rb

```diff
@@ -85,3 +81,20 @@
   assert_match(/SHELL PATH:/, stdout, "PATH expected but not present")
   assert_match(/INSTALLATION DIRECTORY:/, stdout, "GEM_HOME not present")
 end
+
+step "Install hiera-eyaml gem."
+on(master, "#{gem_install} hiera-eyaml --no-ri --no-rdoc")
+
+step "Check that ruby subcommand can load hiera/backend/eyaml from hiera-eyaml"
+on(master, "#{cli} ruby -- -rrubygems -rhiera/backend/eyaml -e 'puts %{OK}'") do
+  assert_match(/^OK$/, stdout, "ruby could not load eyaml after gem install")
+  assert_no_match(/Error/i, stdout, "error loading hiera-eyaml library")
+end
+
+step "Check that irb subcommand can load hiera/backend/eyaml from hiera-eyaml"
```

Comment:
Yeah, I'd normally be fine with a single follow up too.  I included both because we've run into a situation where `gem` behaved differently from the others.  If they're too slow or problematic for some reason we could nuke one of them.